assets/src/common/helpers/test/__mocks__/amp-block-editor-data.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Mock for amp block editor data to be used in featured image test.
+ *
+ * @type {{featuredImageMinimumHeight: number, featuredImageMinimumWidth: number}}
+ */
+export const mockAmpBlockEditorData = {
+	featuredImageMinimumWidth: 1200,
+	featuredImageMinimumHeight: 675,
+};
```

Comment:
Apologies, my last suggestion was mistaken. 

```suggestion
const mockAmpBlockEditorData = {
	featuredImageMinimumWidth: 1200,
	featuredImageMinimumHeight: 675,
};

export default mockAmpBlockEditorData;

```